src/Symfony/Component/Notifier/Bridge/Bluesky/BlueskyTransport.php

```diff
@@ -222,4 +232,51 @@ private function getMatchAndPosition(AbstractString $text, string $regex): array
 
         return $output;
     }
+
+    /**
+     * @param array<array{file: File, description: string}> $media
```

Comment:
What does `File` here refer to? Maybe add the full namespace. 

```suggestion
     * @param array<array{file: \Symfony\Component\Mime\Part\File, description: string}> $media
```

Reply:
Not sure its needed, as it is imported in BlueskyTRansport

Reply:
I did check, but I must have missed it. 

If it is already imported, then please ignore this comment.